publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dynamodb/dao/DynamodbStreamRecordJsonPointers.java

```diff
@@ -37,15 +38,15 @@ public String getValue() {
 
     private static final String ENTITY_DESCRIPTION_REFERENCE_TYPE =
         "/entityDescription/m/reference/m/publicationInstance/m/type/s";
-    private static final String ENTITY_DESCRIPTION_MAP = "/entityDescription/m";
+
```

Comment:
entity description pointer was used only for the publication date and it was replaced.

Reply:
Not at the moment. The Json pointers are basically the physical model hard-coded and we should move away from them because any updating of the physical model (storage model) would mean that we have to go into the code and change the pointers manually. This PR is a fix making the Doi-request chain work.